packages/bruno-electron/src/ipc/network/interpolate-vars.js

```diff
@@ -109,52 +109,32 @@ const interpolateVars = (request, envVars = {}, collectionVariables = {}, proces
   }
 
   if (request?.oauth2?.grantType) {
-    let username, password, scope, clientId, clientSecret;
     switch (request.oauth2.grantType) {
       case 'password':
-        username = _interpolate(request.oauth2.username) || '';
-        password = _interpolate(request.oauth2.password) || '';
-        clientId = _interpolate(request.oauth2.clientId) || '';
-        clientSecret = _interpolate(request.oauth2.clientSecret) || '';
-        scope = _interpolate(request.oauth2.scope) || '';
         request.oauth2.accessTokenUrl = _interpolate(request.oauth2.accessTokenUrl) || '';
-        request.oauth2.username = username;
-        request.oauth2.password = password;
-        request.oauth2.clientId = clientId;
-        request.oauth2.clientSecret = clientSecret;
-        request.oauth2.scope = scope;
-        request.data = {
-          grant_type: 'password',
-          username,
-          password,
-          client_id: clientId,
-          client_secret: clientSecret,
-          scope
-        };
```

Comment:
I am pretty sure the `interpolate-vars` has no business updating `request.data` like this. The `oauth2-helper` knows best what is and what is not to be included in request data, right?